server/mvc/models/User.js

```diff
@@ -1,109 +1,60 @@
+"use strict";
+
+var inherit = require("inherit");
+var ThinkyDocumentModel = require("./ThinkyDocumentModel");
 var Promise = require("bluebird");
 var Cryptr = require("cryptr");
 
-var config,
-  thinky,
-  r,
-  ThinkyModel;
-
-module.exports = function(){
-  var self = this,
-    document = new ThinkyModel({});
+var ThinkyModel;
+var getThinkyModel = function(thinky){
+  if (ThinkyModel === undefined) {
+    ThinkyModel = thinky.createModel("users", {
+      userId: thinky.type.number().integer().default(null).min(1).required().allowNull(false),
+      oauthType: thinky.type.string().default(null).min(1).required().allowNull(false),
+      oauthTokenEncrypted: thinky.type.string().default(null).min(1).required().allowNull(false),
```

Comment:
Yeah. I'll add that. I'm not done with the user flow, yet. All these annoying little things kept bubbling up. :)